.github/workflows/h100-symm-mem.yml

```diff
@@ -0,0 +1,55 @@
+name: Limited CI for symmetric memory tests on H100
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/h100-symm-mem.yml
+  workflow_dispatch:
+  push:
+    tags:
+      - ciflow/h100-symm-mem/*
```

Comment:
You will need to add this into https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml#L33 too

Reply:
oh thanks I forgot this one...